kubernetes-deployment/src/_artifacts.tf

```diff
@@ -0,0 +1,18 @@
+# TODO: remove this artifact
```

Comment:
We can yank this or not. In a world where we might not have 'application artifacts' for a bit, it might be nice to have something to look at, although abusing env-vars might not be the best option.

Maybe we make a debugging artifact that can hold anything?